### PR TITLE
Feature/calendar implementation

### PR DESCRIPTION
**Pull request by zachfedor**
Hit the /fetchevents route to write calendar data to a file. Hit / to read the file data, manipulate it, and send it to the twig template, now with fancy new styles to (hopefully) keep the event tiles in a nice grid.

### CODE REVIEW

File: views/calendar.twig [hunk 1]
```diff
@@ -1 +1,34 @@
-{{ message }}
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Technology,Lancaster,Coding,Programming,Meetup">
```
Comment (amayer5125) [inline views/calendar.twig:7]: This looks more like a keyword list then a description of our website.

Comment (zachfedor) [inline views/calendar.twig:7]: i'm pretty sure that came from [you](https://github.com/TechLancaster/tl-website/commit/17b3e62b45118348978e8feefc729c51ef859f0c), hahaha

Comment (amayer5125) [inline views/calendar.twig:7]: O.o I guess it did... Damn you 2am coding sessions.
If I work on the website this weekend Ill fix it.

Comment (zachfedor) [inline views/calendar.twig:7]: no worries. i laughed a little when i saw it too, i just didn't fix it yet either.

**Pull request merged**